config/sidekiq.yml

```diff
@@ -29,3 +29,8 @@
     class: FetchSpamRepositoriesJob
     queue: default
     description: 'Updates spam repos every 15 minutes'
+  :homepage_project_import:
+    cron: '0 0 * * * America/New_York'
+    class: ProjectImportJob
+    queue: default
+    description: 'Updates the projects displayed on the homepage'
```

Comment:
Very minor, but maybe we should add explanation of chronology since other jobs have it?